reqs/requirements-dev-snowflake.txt

```diff
@@ -1,4 +1,4 @@
 pandas<2.2.0; python_version >= "3.9"
-snowflake-connector-python>=2.5.0; python_version < "3.11"
+snowflake-connector-python>=2.5.0,<3.12.3; python_version < "3.11"
```

Comment:
CI is currently broken due to static-analysis failing on a new version of snowflake connector on python 3.9